packages/angular/cli/README.md

```diff
@@ -164,8 +164,16 @@ You can find more details about changes between versions in [the Releases tab on
 git clone https://github.com/angular/angular-cli.git
 yarn
 npm run build
-cd dist/@angular/cli
-npm link
+(cd dist/@angular/cli;npm link)
```

Comment:
Thinking about this a bit more the npm links should be something like
```
npm link dist/@angular/cli
npm link dist/@angular-devkit/schematics
```

Also, if you are using `@angular/cli` you don't need to use `@angular-devkit/schematics-cli`